File: patterns/javascript-patterns/pattern55.js
// Pattern 55: ...

// 55555
//  ****
//   333
//    **
//     1

const n = 5;

for (let i = n; i >= 1; i--) {
    let row = '';

    for (let j = n; j >= 1; j--) {
        if (i >= j) {
            if (i % 2 === 0) {
                row += '*';
            } else {
                row += i;
            }
        } else {
            row += ' ';
        }
    }

    console.log(row);
}

File: patterns/python-patterns/pattern55.py
# Pattern 55: ...

# 55555
#  ****
#   333
#    **
#     1

n = 5

for i in range(n, 0, -1):
    for j in range(n, 0, -1):
        if i >= j:
            if i % 2 == 0:
                print("*", end="")
            else:
                print(i, end="")
        else:
            print(" ", end="")
    print()
